composer.json

```diff
@@ -20,20 +20,22 @@
             "homepage": "https://github.com/friendsofsymfony/FOSHttpCache/contributors"
         }
     ],
+    "minimum-stability": "dev",
```

Comment:
This `minimum-stability` is necessary because some of the php-http packages themselves require unstable versions of other php-http packages. If your minimum-stability is stable (the default) Composer won’t find them.

When stable versions are available, they will be installed instead: see `"prefer-stable": true` one line below.